pkg/operator/cluster/ceph/mon/health.go

```diff
@@ -126,6 +127,8 @@ func (c *Cluster) checkHealth() error {
 			if time.Since(c.monTimeoutList[mon.Name]) <= MonOutTimeout {
 				logger.Warningf("mon %s NOT found in quorum, STILL in mon out timeout", mon.Name)
 				continue
+			} else {
```

Comment:
This doesn't need to be in the `else` statement, right? It could just be on line 133 before the call to `c.failMon()`